src/ClassicUO.Client/LegionScripting/API.py

```
@@ -0,0 +1,279 @@
+    pass
+
+def Mount(serial):
+    pass
+
+def WaitForTarget(targetType, timeout):
+    pass
+
+def Target(serial):
+    pass
+
+def Target(x, y, z, graphic):
```

Ruff (0.8.2)

Redefinition of unused `Target` from line 160

(F811)